pkg/clusters/utils.go

```diff
@@ -75,3 +75,30 @@ func GetIngressLoadbalancerStatus(ctx context.Context, c Cluster, namespace stri
 		return nil, fmt.Errorf("%T is not a supported ingress type", ingress)
 	}
 }
+
+// CreateNamespace create customized namespace
+func CreateNamespace(ctx context.Context, cluster Cluster, namespace string) error {
+	nsName := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+
+	_, err := cluster.Client().CoreV1().Namespaces().Create(context.Background(), nsName, metav1.CreateOptions{})
+	if err != nil {
+		return fmt.Errorf("failed creating namespace %s, err %v", namespace, err)
+	}
```

Comment:
Related to some previous comments about the license secret name it seems like we would want to tolerate the namespace having already existed:

```suggestion
	_, err := cluster.Client().CoreV1().Namespaces().Create(context.Background(), nsName, metav1.CreateOptions{})
	if err != nil {
	        if errors.IsAlreadyExist(err) {
	            return nil
	        } else {
		    return fmt.Errorf("failed creating namespace %s, err %v", namespace, err)
		}
	}
```

Reply:
We agreed that KTF always deploy new namespace, or clean up existing namespace to avoid error case/error case covering.

Reply:
Added https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-e7bf152c56db96b7d769fe6e6ad4cc4ad16e1c1a42b5558eb793fa38979b1877R91